pkg/kv/kvserver/rangefeed/processor.go

```diff
@@ -734,3 +834,15 @@ func (p *Processor) newCheckpointEvent() *roachpb.RangeFeedEvent {
 	})
 	return &event
 }
+
+// estimateEventSize returns estimated size of the event that contain actual
+// data. We only account for logical ops and sst's for simplicity. Other events
+// are considered infrequent and not large enough.
+func estimateEventSize(e event) int64 {
```

Comment:
If we're not accounting for all event types then we need to be very clear about which ones we budget and which ones we don't, so we don't accidentally forget to release budgets or something. I think I'd prefer that we always account for all event types, which makes it easier to remember that allocations must always be released.

Also, aren't these exact values rather than estimates?

Reply:
I'll rename that to to better reflect what we are doing and why. They are estimates because we don't count overhead of structs containing values. Since we want to protect ourselves from large events those differences should be acceptable.